docs/examples/201/vm-windows-with-custom-script-extension/install.ps1

```diff
@@ -0,0 +1,5 @@
+#Install Chocolatey
+Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
+
+#Install Software
+choco install microsoft-edge obs-studio obs-ndi skype -y
```

Comment:
What is this for?

Reply:
This is the PowerShell script used for custom script extension.

Reply:
As this has a specific intension &ndash; VM provisioning for live streaming, I added it.

Reply:
Gotcha. Yeah, I noticed the URL in the bicep file.

---
In reply to: [481557758](https://github.com/Azure/bicep/pull/425#discussion_r481557758) [](ancestors = 481557758)